README.md

```diff
@@ -1,38 +1,110 @@
+This bot is designed to be a **'ninja helper with no ego'**.
+
+It will ring all unassigned bells perfectly whilst fitting into whatever you are doing, adjusting to
+your changes in rhythm.
+It will never try to take control of the ringing by giving instructions or powering on with a
+different rhythm to everyone else (unless you tell it to).
+However, by default it will adjust only to changes to overall rhythm - it will ignore individual
+bells holding up and ring in the 'right' place regardless.
+
+Only one ringer needs to run the command and it will ring with you after anyone calls `Look to` in
+Ringing Room.
+It understands `Go`, `Look to`, `That's all` and `Stand next` in Ringing Room, but currently can
+only ring plain courses of any given method (or touches from CompLib).
+
+### Notable features
+- Configurable rhythm detection that updates in real time
+- Automatic correction for Ringing Room using multiple server URLs for load balancing
+- Very light resource footprint, as it doesn't interact with a browser to talk to Ringing Room
+
+### Roadmapped features
+- Make the bot able to ring touches of methods
 
 ## Quickstart
 ### Installation (same for all platforms if using the command line)
-1. Clone and download this repository.
+1. Clone and download this repository (using
+   [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) will make updating easier):
+   ```bash
+   git clone https://github.com/Kneasle/ringing-room-bot
+   ```
+   (you can update the bot at any time with `git pull origin master`)
 
 2. Make sure you have Python 3.x installed with:
    ```bash
    python3 --version
    ```
-   If Python 3.x is installed it will print the version and if python 3.x isn't installed it will produce an error.
-   
+   If Python 3.x is installed it will print the version and if Python 3.x isn't installed it will
+   produce an error.
+
 3. Move to the location of the repository:
    ```bash
-   cd <repository location>
+   cd [REPOSITORY LOCATION]
    ```
 
 4. Install the required python packages:
    ```bash
    pip3 install -r requirements.txt
    ```
-5. Start the bot with the following command, replacing `Plain Bob Major` with the title of the method you want to ring:
-   ```bash
-   python3 main.py --id <ringing room tower id> --method "Plain Bob Major"
-   ```
-On Unix you can run main.py directly:
-```bash
-./main.py --id <ringing room tower id> --method "Plain Bob Major"
-```
 
-For a list of options, you can run the bot with the `--help` or `-h` argument.
+5. Pick an example below to run the bot!
+
+## Examples
+Run the bot with `python3 main.py [args]` (or `./main.py [args]` on Unix).
+You may have to use `python` instead of `python3` on Windows.
+
+*   Join a `ringingroom.com` tower with (9 digit) id `[ID NUMBER]` and ring Plain Bob Major (tower
+    bell style):
+    ```bash
+    python3 main.py --id [ID NUMBER] --method "Plain Bob Major"
```

Comment:
Possibly something that make this clearer this is the number of tower in ringing room
`[TOWER ID]` ?
`[RINGING ROOM ID]` ?

Reply:
So I was worried that people would think that the name of the tower is its tower ID, whereas `[ID NUMBER]` is explicit about this.